packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -796,7 +798,7 @@ export default class DCWebxdc {
         }
       }
     )
-  }
+  } // end of DeltaChatController constructor
```

Comment:
The constructor is very long and it's hard to know which scope you're in while reading this code.

Reply:
there is also `// #region` `// #endregion` folding support in editors like vscode/ium

Reply:
I think I'd like to break this constructor apart a bit later anyway.